cypress/e2e/tests/pages/explorer/cluster-dashboard.spec.ts

```diff
@@ -0,0 +1,171 @@
+    clusterDashboard.clusterBadge().selectCheckbox('Customize cluster icon').set();
+    clusterDashboard.clusterBadge().clusterIcon().children().should('not.have.class', 'cluster-local-logo');
+    clusterDashboard.clusterBadge().clusterIcon().contains(settings.iconText.original);
+    clusterDashboard.clusterBadge().iconText().set(settings.iconText.new);
+    clusterDashboard.clusterBadge().clusterIcon().contains(settings.iconText.new);
+
+    // Apply Changes
+    clusterDashboard.clusterBadge().applyAndWait('/v3/clusters/local');
+
+    // check header and side nav for update
+    header.clusterIcon().children().should('have.class', 'cluster-badge-logo');
+    header.clusterName().should('contain', 'local');
+    header.clusterBadge().should('contain', settings.description.new);
+    const burgerMenu = new BurgerMenuPo();
+
+    burgerMenu.clusters().first().find('span').should('contain', settings.iconText.new);
+
+    // Reset
+    clusterDashboard.addClusterBadge('Edit Cluster Badge').click();
+    clusterDashboard.clusterBadge().selectCheckbox('Customize cluster icon').set();
+    clusterDashboard.clusterBadge().selectCheckbox('Show badge for this cluster').set();
+
+    // Apply Changes
+    clusterDashboard.clusterBadge().applyAndWait('/v3/clusters/local');
+
+    // check header and side nav for update
+    header.clusterIcon().children().should('have.class', 'cluster-local-logo');
+    header.clusterName().should('contain', 'local');
+    header.clusterBadge().should('not.exist');
+    burgerMenu.clusters().first().find('svg').should('have.class', 'cluster-local-logo');
+  });
+
+  it('can view deployments', () => {
+    clusterDashboard.goTo();
+    cy.getRancherResource('v1', 'apps.deployments', '?exclude=metadata.managedFields').then((resp: Cypress.Response<any>) => {
+      const count = resp.body['count'];
+
+      simpleBox.simpleBox().eq(2).should('contain.text', count).and('contain.text', 'Deployments');
+    }).then((el: any) => {
+      el.click();
+
+      const workloadDeployments = new WorkloadsDeploymentsListPagePo('local', 'apps.deployment');
+
+      workloadDeployments.waitForPage();
+    });
+  });
+
+  it('can view nodes', () => {
+    clusterDashboard.goTo();
+    cy.getRancherResource('v1', 'nodes', '?exclude=metadata.managedFields').then((resp: Cypress.Response<any>) => {
+      const count = resp.body['count'];
+      let text = '';
+
+      if (count > 1) {
+        text = 'Nodes';
+      } else {
+        text = 'Node';
+      }
+      simpleBox.simpleBox().eq(1).should('contain.text', count).and('contain.text', text);
+    }).then((el: any) => {
+      el.click();
+
+      const workloadNodes = new WorkloadsDeploymentsListPagePo('local', 'node');
+
+      workloadNodes.waitForPage();
+    });
+  });
+
+  it('can view events', () => {
+    clusterDashboard.goTo();
+
+    // Install monitoring to trigger events
+    clusterDashboard.clusterToolsButton().click();
```

Comment:
To create some events it might be safer to create a pod instead of installing a chart. This means during development running them against a system that might already have the chart installed would be fine (and won't uninstall from that system either)

Reply:
good call. I've updated the test to create a pod (via api; I did this since we're not testing that area of the application but rather events) instead of installing a chart.